homeassistant/components/weatherflow_cloud/sensor.py

```diff
@@ -0,0 +1,239 @@
+        translation_key="air_temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.observation.obs[0].air_temperature,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="dew_point",
+        translation_key="dew_point",
+        value_fn=lambda data: data.observation.obs[0].dew_point,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="feels_like",
+        translation_key="feels_like",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.observation.obs[0].feels_like,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="heat_index",
+        translation_key="heat_index",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.observation.obs[0].heat_index,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="wind_chill",
+        translation_key="wind_chill",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.observation.obs[0].wind_chill,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="wet_bulb_temperature",
+        translation_key="wet_bulb_temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.observation.obs[0].wet_bulb_temperature,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="wet_bulb_globe_temperature",
+        translation_key="wet_bulb_globe_temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.observation.obs[0].wet_bulb_globe_temperature,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        extra_state_attributes_fn=lambda data: {
+            "flag": data.observation.obs[0].wet_bulb_globe_temperature_flag.name,
+            "category": data.observation.obs[0].wet_bulb_globe_temperature_category,
+        },
```

Comment:
We like to avoid adding extra state attributes as much as possible, can you explain what these tell us?